apps/src/componentLibrary/tabs/tabs.module.scss

```diff
@@ -0,0 +1,320 @@
+@import 'color';
+@import 'font';
+@import "@cdo/apps/componentLibrary/common/styles/mixins";
+
+.tabs {
+  ul {
+    display: inline-flex;
+    align-items: flex-start;
+    margin: 0;
+    padding: 0;
+
+    li {
+      list-style: none;
+      margin: 0;
+      padding: 0;
+    }
+  }
+
+  button {
```

Comment:
Out of curiosity, why not use class names here?

Reply:
Not using classNames here since it's not required.(html structure of this component is pretty basic) Keeping css selectors priority as low as possible in order make things simple.